documentation/docs/exchanges/bitmart.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+- spot exchange connector
+---
+
+# `bitmart`
+
+## 📁 [Connector folder](https://github.com/CoinAlpha/hummingbot/tree/development/hummingbot/connector/exchange/bitmart)
+
+## ℹ️ Exchange Info
+
+**BitMart**
+[Website](https://www.bitmart.com/) | [CoinMarketCap](https://coinmarketcap.com/exchanges/bitmart/) | [CoinGecko](https://www.coingecko.com/en/exchanges/bitmart)
+
+- API docs: https://developer-pro.bitmart.com/en/
+- Transaction fees: https://www.bitmart.com/fee/en
+- Minimum order size: Equivalent value of 5 USDT
```

Comment:
If we don't have a source for the minimum order, then leave it blank.

Reply:
Removed the minimum order size, I can't find any link that shows this on BitMart.